### ISSUE TITLE
[BUG] Env variables deprecation in latest RabbitMQ version

### ISSUE THREAD

**Issue by Muarline** [taigaio/taiga-docker#34]
**Describe the bug**

RabbitMQ deprecated in the `3.9` release a bunch of environment variables that were used by the `taiga-docker` repository. Containers `taiga-async-rabbitmq` and `taiga-events-rabbitmq` fail to start because of this change. Both `stable` and `main` branches are affected.

**How can we reproduce the behavior**

Clone the repo from `stable` or `main` branch. Run `./launch-taiga.sh`. Both containers will exit with an error.

**Workarounds**

Specify a concrete RabbitMQ version in the image tag (e.g. `rabbitmq:3-management-alpine` -> `rabbitmq:3.8-management-alpine`).

**Desktop (please complete the following information):**
 - **OS:** Arch Linux

**Comment by curiv** [comment]
Muarline's workaround worked for me.

But still on /discover page I see:
_Something happened and the Taiga has captured the error to be able to work on it._

And no logs after it

**Comment by regzon** [comment]
@hesoyamimi 
Strange, we've applied this fix and Taiga works fine now on every page.
Are other containers alive? What do you mean by "no logs"? Like, no logs in the browser client, or in all docker containers?

**Comment by curiv** [comment]
> @hesoyamimi
> Strange, we've applied this fix and Taiga works fine now on every page.
> Are other containers alive? What do you mean by "no logs"? Like, no logs in the browser client, or in all docker containers?

I just figured that out. Previously there were no logs in docker. Before the first launch I had an error about docker-compose version. I downgraded it to 2.2 and it worked untill web app started.

After It I decided to update docker-compose package to the latest version (current was pretty old). After it I received no errors and http interface just loaded fine.
